Public/Src/Cache/DistributedCache.Host/Service/Internal/DualServerClientContentStore.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.ContractsLight;
+using System.Threading;
+using System.Threading.Tasks;
+using BuildXL.Cache.ContentStore.Distributed.Redis.Credentials;
+using BuildXL.Cache.ContentStore.Interfaces.Distributed;
+using BuildXL.Cache.ContentStore.Interfaces.FileSystem;
+using BuildXL.Cache.ContentStore.Interfaces.Logging;
+using BuildXL.Cache.ContentStore.Interfaces.Results;
+using BuildXL.Cache.ContentStore.Interfaces.Sessions;
+using BuildXL.Cache.ContentStore.Interfaces.Stores;
+using BuildXL.Cache.ContentStore.Interfaces.Tracing;
+using BuildXL.Cache.ContentStore.Service;
+using BuildXL.Cache.ContentStore.Sessions;
+using BuildXL.Cache.ContentStore.Stores;
+using BuildXL.Cache.Host.Configuration;
+
+namespace BuildXL.Cache.Host.Service.Internal
+{
+    /// <summary>
+    /// Content store containing both a distributed <see cref="LocalContentStore"/> and <see cref="ServiceClientContentStore"/>.
+    /// </summary>
+    public class DualServerClientContentStore : IContentStore
+    {
+        private LocalContentServer _localContentServer;
+        private ServiceClientContentStore _serviceClientContentStore;
+
+        /// <inheritdoc />
+        public bool StartupCompleted => (_localContentServer?.StartupCompleted ?? false) && (_serviceClientContentStore?.StartupCompleted ?? false);
+
+        /// <inheritdoc />
+        public bool StartupStarted => (_localContentServer?.StartupStarted ?? false) && (_serviceClientContentStore?.StartupStarted ?? false);
+
+        /// <inheritdoc />
+        public bool ShutdownCompleted => (_localContentServer?.ShutdownCompleted ?? false) && (_serviceClientContentStore?.ShutdownCompleted ?? false);
+
+        /// <inheritdoc />
+        public bool ShutdownStarted => (_localContentServer?.ShutdownStarted ?? false) && (_serviceClientContentStore?.ShutdownStarted ?? false);
+
+        /// <summary>
+        /// 
+        /// </summary>

```

Comment:
nodoc is fine here, I think.
Or fill the summary.

Reply:
Whoops! Missed this one.

---
In reply to: [277808204](https://github.com/Microsoft/BuildXL/pull/185#discussion_r277808204) [](ancestors = 277808204)

Reply:
But since we're moving to use configurations, documentation would be wasted here.

---
In reply to: [277815496](https://github.com/Microsoft/BuildXL/pull/185#discussion_r277815496) [](ancestors = 277815496,277808204)